manifests/s/SABnzbdTeam/SABnzbd/Potable/4.2.3/SABnzbdTeam.SABnzbd.Potable.yaml

```
@@ -0,0 +1,8 @@
+# Created with komac v2.2.1
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.version.1.6.0.schema.json
+
+PackageIdentifier: SABnzbdTeam.SABnzbd.Potable

```

```suggestion
PackageIdentifier: SABnzbdTeam.SABnzbd.Portable
```
Please update file name and path to match this change.